getting-started/initialize-the-sdk.md

```diff
@@ -103,6 +103,56 @@ ACPCore.setLogLevel(ACPMobileLogLevel.debug)
 {% endtab %}
 {% endtabs %}
 
+## Registering Extensions and Starting Core
+
+All AEP extensions provide a `registerExtension` API (except Core), this API registers the extension with Core which allows the extension to dispatch and listen for events. You are required to register each of your extensions before making API calls and failing to do so will lead to undefined behavior.
+
+Once you have all your extensions registered you must call the `start` API in Core. This is required to boot up the SDK for event processing.
```

Comment:
Please change the sentence so that it reads as follows:

"After you register all of your extensions, call the `start` API in Core. This step is required to boot up the SDK for event processing."